src/test/java/gustavohuanca/code/typeCall/CallPost.java

```diff
@@ -0,0 +1,32 @@
+package gustavohuanca.code.typeCall;
+
+import gustavohuanca.code.SettingParameter;
+import gustavohuanca.code.entities.Epic;
+import io.restassured.http.ContentType;
+
+import static io.restassured.RestAssured.given;
+
+public class CallPost implements IRequestCall{
+    private Epic epic;
+
+    public CallPost(Epic epic){
+        this.epic=epic;
+    }
+
+    @Override
+    public String answerToCall(String pathEndPoint) {
+        return responseCallStatus(pathEndPoint)+"";
+    }
+
+
+    private int responseCallStatus(String pathEndPoint){
+        return given()
+                .header(SettingParameter.getTokenKey(), SettingParameter.getTokenValue())
+                .contentType(ContentType.JSON)
+                .body(epic)
```

Comment:
this solution is interesting but is to fully customizable. for example this is going to work only for epics